geowebcache/mbtiles/src/main/java/org/geowebcache/mbtiles/layer/MBTilesCacheInfo.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * <p>You should have received a copy of the GNU Lesser General Public License along with this
+ * program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * <p>Copyright 2021
+ */
+package org.geowebcache.mbtiles.layer;
+
+import static org.geotools.mbtiles.MBTilesFile.SPHERICAL_MERCATOR;
+import static org.geotools.mbtiles.MBTilesFile.WORLD_ENVELOPE;
+
+import java.io.IOException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.mbtiles.MBTilesFile;
+import org.geotools.mbtiles.MBTilesMetadata;
+import org.geotools.referencing.CRS;
+import org.geowebcache.grid.BoundingBox;
+import org.opengis.geometry.Envelope;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.crs.CoordinateReferenceSystem;
+import org.opengis.referencing.operation.TransformException;
+
+/** Info Object storing basic MBTiles Cached info */
+public class MBTilesCacheInfo {
```

Comment:
```suggestion
public class MBTilesInfo {
```

Rationale: this class collects metadata from the MBTiles file. Whether it's cached or not, it's a  matter of how it's used, it could be re-instantiated at every need too. 

If the instance variable in MBTilesLayer was a cache the field name holding it could be called that way instead.

Reply:
The field was named cacheInfo.
I named that class like that to avoid "confusion" with the MbtilesInfo of sqllite.
I will rename it.